server/src/test/java/org/elasticsearch/search/query/QueryPhaseCollectorTests.java

```diff
@@ -102,6 +102,8 @@ public void testNegativeTerminateAfter() {
         );
     }
 
+    // TODO remove some of these tests as they are now duplicated?
+
```

Comment:
I wonder what your opinion is on test coverage: we are duplicating tests right now, is it still needed to test sequential execution explicitly that does not go through the manager? Some of these existing tests are very specific to the collector which I am going to keep, but the ones that I copied to the new test class and modified to use a collector manager could be removed? I am not entirely sure.

Reply:
I didn't take a closer side-by-side look yet, but any methods you are considering specifically?

Reply:
> the ones that I copied to the new test class and modified to use a collector manager could be removed?

Reply:
I have moved over all the search tests to using the collector manager. The collector low-level tests that don't do a search still test the collector directly.